spec/models/account_spec.rb

```diff
@@ -147,10 +147,10 @@
   it "should set suspend_at on suspend and unsuspend" do
     @owner   = FactoryGirl.create(:user)
     @account = FactoryGirl.create(:nufs_account, account_users_attributes: account_users_attributes_hash(user: @owner))
-    assert_nothing_raised { @account.suspend! }
-    expect(@account.suspended_at).not_to eq(nil)
-    assert_nothing_raised { @account.unsuspend! }
-    expect(@account.suspended_at).to eq(nil)
+    expect(@account.suspend).to be_truthy
```

Comment:
`eq(true)` is more likely to catch a bug if one pops up, but overall it's not a big deal.  :+1: